.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Please note this is an issue tracker, not a support forum.
+For general questions, please use StackOverflow or Slack.
+-->
+
+## What are you doing?
+<!-- Post a minimal, self-contained code sample that reproduces the issue, including models and associations -->
+
+```js
+// code here
+```
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Define models X, Y, ...
+2. Run the following
+3. See error
+
+## What do you expect to happen?
+_I wanted Foo!_
+
+## What is actually happening?
+_But the output was bar!_
+
+_Output, either JSON or SQL_
+
+## Environment
+Dialect:
```

Comment:
Please also add `OS` and Node version.
Also in case of typescript errors we should ask for the typescript version as well.